app/src/main/res/values/strings.xml

```diff
@@ -282,6 +282,8 @@
     <string name="preferences_theme">Theme</string>
     <!-- Preference for customizing the home screen -->
     <string name="preferences_home">Home</string>
+    <!-- Preference for gestures based actions -->
+    <string name="preferences_gestures">Gestures</string>
```

Comment:
@betsymi could you confirm these strings? :)

Reply:
Looks good.